### PR TITLE
Adds missing license attributes

### PR DESCRIPTION
**Pull request by lightsprint09**
This repo is missing license information. It would be great if license information could be added. MIT is just a guess.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,6 +12,13 @@
 	<packaging>hpi</packaging>
 	<name>Hudson Post build task</name>
 	<version>1.9-SNAPSHOT</version>
+	<licenses>
+    		<license>
+      			<name>The MIT license</name>
+      			<url>http://www.opensource.org/licenses/mit-license.php</url>
+     			<distribution>repo</distribution>
```
Comment (oleg-nenashev) [inline pom.xml:19]:
```suggestion
      			<url>https://opensource.org/licenses/MIT</url>
     			<distribution>repo</distribution>
```

**Review by oleg-nenashev — COMMENTED** [review]
An entry would not hurt, but it should point to HTTPs. Also, there is a mix of tables vs. spaces

// Just passing by in this repo

**Review by timja — APPROVED** [review]
I fixed the indentation in the file and did minor cleanup

**Comment by lightsprint09** [comment]
Thanks 👍

**Pull request merged**